src/mindlessgen/prog/config.py

```diff
@@ -180,6 +180,7 @@ def __init__(self: GenerateConfig) -> None:
         self._forbidden_elements: list[int] | None = None
         self._scale_vdw_radii: float = 4.0 / 3.0
         self._scale_minimal_bondlength: float = 0.75
```

Comment:
```suggestion
        self._scale_vdw_radii: float = 1.25
        self._scale_minimal_bondlength: float = 0.8
```
What do you think about adjusting the defaults?

Reply:
This looks more convenient.